scripts/generate_package_docs.py

```diff
@@ -21,7 +21,7 @@ def get_docs(params):
 
   docs = {}
   for tool in tools:
-    print "Writing doc for tool {}".format(tool)
+    print ("Writing doc for tool {}".format(tool))
```

Comment:
don't know python 2 quite well, would this work in python2? (I assume Jenkins job still uses python2)

Reply:
@jdcanas